.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,30 @@
+---
+name: Bug report
+about: Create a report to help us improve `php-github-api`
+
+---
+
+**Configuration (preferably required)**
+- **php-github-api version:** "...replace me..."
+- **PHP version:** "...replace me (Can be retrieved using command line `php -v | sed -n 1p`)..."
+- **Operating system:** " ...replace me (You REALLY should know it)... "
```

Comment:
Not needed. I've rarely seen an issue on this repo when this matter.

Reply:
Ok, I'll remove it.